pymtl3/passes/rtlir/behavioral/BehavioralRTLIRImplGen.py

```diff
@@ -185,7 +182,7 @@ def impl_str( s ):
 
       params_name = ', ' + ', '.join( s.field_list )
       params_assign = '\n    '.join(
-        map( lambda x: "s.{field} = {field}".format(field = x), s.field_list )
+        ["s.{field} = {field}".format(field = x) for x in s.field_list]
```

Comment:
This would be more idiomatic without the `[]`, since you can pass a generator expression without having to construct the intermediate list, and also slightly faster.